pkg/northbound/gnmi/extensions.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gnmi
+
+const (
+	// GnmiExtensionNetwkChangeID is the extension number used in SetRequest and SetResponse
```

Comment:
we should have a document in this patch to specify what these flags are for and especially what happens if you do not specify them.

Reply:
I think there will have to be a table of what they mean in run.md - I was going to prepare it for the TST tomorrow, so it will be good prep

Reply:
+1